required/tools/varioref.dtx

```diff
@@ -1036,7 +1036,7 @@
     \let\vrefrangeformat\vrefrangedefaultformat
   }}
 %    \end{macrocode}
-%    Default string for dutch have been contributed by Frank Poppe
+%    Default strings for dutch have been contributed by Frank Poppe
```

Comment:
I guess it would be correct to use always uppercase for language names. That is just lousy/lazy writing on my parts (or maybe copying in what others sent me)

Reply:
Easy to get this wrong!
The rule is, I think, clear when it is a noun, but using "french" as an adjective only sometimes seems to deserve Uppercase. 

And then there are cases when it is unclear whether it is being used as a noun or as an adjective!